pg_search/src/index/merge_policy.rs

```diff
@@ -4,12 +4,6 @@ use pgrx::pg_sys;
 use tantivy::indexer::{MergeCandidate, MergePolicy};
 use tantivy::SegmentMeta;
 
-#[derive(Debug, Clone)]
-pub enum AllowedMergePolicy {
-    None,
-    NPlusOne(usize),
-}
-
```

Comment:
We discussed offline about actually keeping this, but with **only** the `None` variant.

That'll let us more easily change our minds in the future without a changeset that touches mostly function signatures.  And will clean this one up too.

I know it's an annoying nit, but I think it's worth the quick rework here.